src/findex.rs

```diff
@@ -265,4 +261,83 @@ mod tests {
             res
         );
     }
+
+    #[test]
+    fn test_insert_search_delete_search_redis() {
```

Comment:
no redis-specific test in the main file : either make it backend-independent or move it to some redis-related module.